src/lib.rs

```diff
@@ -161,7 +168,7 @@ pub fn from_path(p: &str) -> Result<String> {
         at_root_has_selector: false,
         extender: &mut Extender::new(empty_span),
         content_scopes: &mut Scopes::new(),
-        load_paths: &Vec::new(),
+        load_paths: loadpaths,
```

Comment:
You could keep `loadpaths` as `load_paths` and this could be
```suggestion
        load_paths,
```